backend/Gemfile

```diff
@@ -19,14 +19,14 @@ gem "rufus-scheduler", "~> 2.0.24"
 gem "rufus-lru", "1.0.5"
 gem "net-ldap", '~> 0.16.0'
 gem "mizuno", "0.6.11"
-gem "i18n", "0.7.0"
+gem "i18n", "0.9.1"
+gem "pry-debugger-jruby"
```

Comment:
@avatar382 Can you put the pry-debugger-jruby gem inside a `group :development, :test` block?